lib/elixir/src/elixir_dispatch.erl

```diff
@@ -365,6 +365,12 @@ check_deprecated(Meta, 'Elixir.System', stacktrace, 0, #{contextual_vars := Vars
           "move System.stacktrace/0 inside a rescue/catch",
       elixir_errors:erl_warn(?line(Meta), ?key(E, file), Message)
   end;
+
+check_deprecated([{no_parens,true}, _] = Meta, _, _, 0, E) ->
```

Comment:
We wan't to do this warning here instead https://github.com/elixir-lang/elixir/blob/master/lib/elixir/src/elixir_expand.erl#L404.